### PR TITLE
Fixed wrong code example in ArrayField documentation

### PR DESCRIPTION
**Pull request by agmond**

### CODE REVIEW

File: docs/ref/contrib/postgres/fields.txt [hunk 1]
```diff
@@ -222,7 +222,7 @@ lookups available after the transform do not change. For example::
     >>> Post.objects.create(name='Third post', tags=['django', 'python', 'thoughts'])
 
     >>> Post.objects.filter(tags__0_1=['thoughts'])
-    <QuerySet [<Post: First post>]>
+    <QuerySet [<Post: First post>, <Post: Second post>]>
 
     >>> Post.objects.filter(tags__0_2__contains='thoughts')
```
Comment (timgraham) [inline docs/ref/contrib/postgres/fields.txt]: Noticed this one also fails. Do you think it's a bug or a bad example?
```
django.db.utils.DataError: malformed array literal: "thoughts"
LINE 1: ...extras_post" WHERE "pgextras_post"."tags"[1:2] @> 'thoughts'...
```

Comment (agmond) [inline docs/ref/contrib/postgres/fields.txt]: I think that this example can be fixed by surrounding the 'thoughts' argument with square brackets:
```
>>> Post.objects.filter(tags__0_2__contains=['thoughts'])
```

I've updated the PR accordingly.

**Comment by timgraham** [comment]
merged in a44dc200d0cbd651f66a2082aa9909ff8f51a659, thanks!

**Comment by shaib** [comment]
:+1: :cake:

**Pull request closed without merging**